mne/decoding/tests/test_time_gen.py

```diff
@@ -424,4 +424,23 @@ def test_decoding_time():
     tg = TimeDecoding(clf=clf, cv=cv)
     tg.fit(epochs, y=y)
 
+
+def test_string_scorer():
+    """Test whether passing string as scorer works as intended"""
+    epochs = make_epochs()
```

Comment:
I am sorry, but I don't understand this clearly. If I slice epochs to `epochs=epochs[:2]` and then set `cv=2` then I get the following error.
```
ValueError                                Traceback (most recent call last)
<ipython-input-43-f8251ac2533f> in <module>()
----> 1 gat.fit(epochs[:2])

/home/kaichogami/codes/dev_mne/mne-python/mne/decoding/time_gen.pyc in fit(self, epochs, y)
    140         # Prepare cross-validation
    141         self.cv_, self._cv_splits = _set_cv(self.cv, clf=self.clf, X=X, y=y)
--> 142 
    143         self.y_train_ = y
    144 

/home/kaichogami/codes/dev_mne/mne-python/mne/decoding/time_gen.pyc in _set_cv(cv, clf, X, y)
   1514 
   1515     if not np.all([len(train) for train, _ in cv_splits]):
-> 1516         raise ValueError('Some folds do not have any train epochs.')
   1517 
   1518     return cv, cv_splits

ValueError: Some folds do not have any train epochs.
```
It takes minimum 4 events to work properly.